samples/dungeon/timemachine/src/test/java/com/squareup/sample/timemachine/TimeMachineWorkflowTest.kt

```diff
@@ -25,7 +25,7 @@ class TimeMachineWorkflowTest {
     val delegateWorkflow = Workflow.stateful<String, Nothing, DelegateRendering>(
       initialState = "initial",
       render = { renderState ->
-        DelegateRendering(renderState, setState = eventHandler { s -> state = s })
+        DelegateRendering(renderState, setState = eventHandler("") { s -> state = s })
```

Comment:
Should we have a `require(!name.isEmpty())` or is that just mean/unecessary?

Reply:
I'm still torn on this. We definitely shouldn't be throwing exceptions for empty output from the `name: () -> String` variants, that seems like it would be a major landmine. And really, even the static string version seems ripe to blow up only after release -- you enter a a state that is less common, you forgot to unit test it, and suddenly an obscure feature is blowing up. I don't think we should do it.